src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -137,6 +174,16 @@ export function transform(formConfig, form) {
 
   const attachments = additionalDocuments.concat(privateRecords);
 
+  const providerFacility = disabilities
```

Comment:
This is actually an array of (potentially) several provider facilities, right?

Reply:
Correct

Reply:
To avoid confusion, what do you think about using plurals for arrays? `providerFacilities` would then contain (potentially) many provider facilities.

Reply:
@cvalarida I can definitely agree with that, the only problem is the Schema, and backend around it are `providerFacility` so changing the variable name would be the only place we could do it without a lot of overhead. I've made the change to the variable specifically. In the future I'll advise the team that we try to stick to this naming convention to avoid confusion.